lib/cfg-tree.c

```diff
@@ -1065,6 +1066,38 @@ cfg_tree_compile(CfgTree *self)
   return TRUE;
 }
 
+static gboolean
+verify_unique_persist_ids_among_pipes(const GPtrArray * initialized_pipes)
+{
+  GHashTable * pipe_persist_ids = g_hash_table_new(g_str_hash, g_str_equal);
+  gboolean     result           = TRUE;
+
```

Comment:
In syslog-ng code base we don't use this kind of vertical alignment.
and we bind the pointer to the variable (` *initialized_pipes` and later, inside the for cycle, ` *current_pipe` is suggested). 
We've talked about this ;-)